EasyPost.Tests/EasyPost.Tests.csproj

```diff
@@ -31,15 +31,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="EasyVCR" Version="0.5.1"/>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="[17.3.0, 18.0.0)"/>
+    <PackageReference Include="EasyVCR" Version="0.5.1" />
```

Comment:
If I see your IDE adjust these spaces in one more PR, I'm gonna lose it 😂 

Please fix whatever is doing this to stop doing it. It literally flips back and forth every time these files get touched.